module-utils/log/Logger.cpp

```diff
@@ -1,28 +1,28 @@
 // Copyright (c) 2017-2022, Mudita Sp. z.o.o. All rights reserved.
 // For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
 
-#include "critical.hpp"
-#include <fstream>
+#include "Logger.hpp"
 #include "LockGuard.hpp"
-#include <Logger.hpp>
-#include <Utils.hpp>
-#include <portmacro.h>
 #include <ticks.hpp>
-#include "macros.h"
+#include <purefs/filesystem_paths.hpp>
+
+#include <fstream>
 
 namespace Log
 {
     const char *Logger::levelNames[] = {"TRACE", "DEBUG", "INFO", "WARN", "ERROR", "FATAL"};
     Logger *Logger::_logger          = nullptr;
 
+    constexpr std::chrono::milliseconds writeLogsToFileInterval{60 * 1000}; // TODO: for testing 60s, normally 15 min
```

Comment:
you can use proper container (minutes) instead of ms - you can then do the `duration_cast` ;)